tools/nnyeah/nnyeah/Reworker.cs

```diff
@@ -257,6 +268,24 @@ void ReworkCodeBlock (MethodBody body)
 		{
 			var changes = new List<Tuple<Instruction, Transformation>> ();
 			foreach (var instruction in body.Instructions) {
+				var operandText = instruction.Operand?.ToString ();
+				if (operandText is not null) {
+					if (moduleMap.TypeIsNotPresent (operandText)) {
+						throw new TypeNotFoundException (operandText);
+					}
+					if (moduleMap.MemberIsNotPresent (operandText)) {
+						throw new MemberNotFoundException (operandText);
+					}
+					if (moduleMap.TryGetMappedType (operandText, out var type)) {
+						var newInstruction = ChangeTypeInstruction (instruction, type);
+						changes.Add (new Tuple<Instruction, Transformation> (instruction, new Transformation (operandText, newInstruction)));
+						continue;
+					} else if (moduleMap.TryGetMappedMember (operandText, out var member)) {
+						var newInstruction = ChangeMemberInstruction (instruction, member);
+						changes.Add (new Tuple<Instruction, Transformation> (instruction, new Transformation (operandText, newInstruction)));
+						continue;
```

Comment:
Looks like we do the same line twice:
```csharp
changes.Add (new Tuple<Instruction, Transformation> (instruction, new Transformation (operandText, newInstruction)));
```
Can we rewrite it so that we get to write this a single time?

Reply:
Not cleanly. Types and members are two different things.